src/main/kotlin/io/csbroker/batchserver/config/BatchConfig.kt

```diff
@@ -0,0 +1,144 @@
+package io.csbroker.batchserver.config
+
+import io.csbroker.batchserver.client.AIServerClient
+import io.csbroker.batchserver.dto.GradingRequestDto
+import io.csbroker.batchserver.dto.GradingResponseDto
+import io.csbroker.batchserver.entity.GradingHistory
+import io.csbroker.batchserver.entity.GradingStandard
+import io.csbroker.batchserver.util.log
+import org.springframework.batch.core.ExitStatus
+import org.springframework.batch.core.Job
+import org.springframework.batch.core.Step
+import org.springframework.batch.core.StepExecution
+import org.springframework.batch.core.StepExecutionListener
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory
+import org.springframework.batch.core.configuration.annotation.JobScope
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory
+import org.springframework.batch.core.configuration.annotation.StepScope
+import org.springframework.batch.item.ItemProcessor
+import org.springframework.batch.item.database.JpaItemWriter
+import org.springframework.batch.item.database.JpaPagingItemReader
+import org.springframework.batch.item.database.builder.JpaItemWriterBuilder
+import org.springframework.batch.item.database.builder.JpaPagingItemReaderBuilder
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.time.LocalDateTime
+import javax.persistence.EntityManagerFactory
+
+@Configuration
+class BatchConfig(
+    private val jobBuilderFactory: JobBuilderFactory,
+    private val stepBuilderFactory: StepBuilderFactory,
+    private val entityManagerFactory: EntityManagerFactory,
+    private val aiServerClient: AIServerClient
+) {
+
+    @Bean
+    fun gradingJob(): Job {
+        return jobBuilderFactory.get("reGradingJob")
+            .start(gradingStep())
+            .build()
+    }
+
+    @Bean
+    @JobScope
+    fun gradingStep(): Step {
+        return stepBuilderFactory.get("reGradingStep")
+            .chunk<GradingHistory, GradingHistory>(10)
+            .reader(reader(0L, ""))
+            .processor(processor(0L, ""))
+            .writer(writer(0L, ""))
+            .listener(stepExecutionListener())
+            .build()
+    }
+
+    @Bean
+    @StepScope
+    fun reader(
+        @Value("#{jobParameters[problemId]}") problemId: Long,
+        @Value("#{jobParameters[date]}") date: String
+    ): JpaPagingItemReader<GradingHistory> {
+        log.info("==> reader date $date, problem id : $problemId")
+
+        val parameterValues = mutableMapOf<String, Any>()
```

Comment:
```suggestion
        val parameterValues = mutableMapOf<String, Long>()
```
Any 보단 명확한 타입을 적는게 좋을 것 같아요!